cookbooks/message_cookbook.ipynb

```diff
@@ -0,0 +1,807 @@
+            "Requirement already satisfied: omegaconf>=2.0 in /usr/local/lib/python3.10/dist-packages (from effdet->layoutparser[layoutmodels,tesseract]->unstructured-inference==0.7.11->unstructured[all-docs]<0.11,>=0.10; extra == \"tools\" or extra == \"all\"->camel-ai[all]==0.1.6.8) (2.3.0)\n",
+            "Requirement already satisfied: pypdfium2>=4.18.0 in /usr/local/lib/python3.10/dist-packages (from pdfplumber->layoutparser[layoutmodels,tesseract]->unstructured-inference==0.7.11->unstructured[all-docs]<0.11,>=0.10; extra == \"tools\" or extra == \"all\"->camel-ai[all]==0.1.6.8) (4.30.0)\n",
+            "Requirement already satisfied: antlr4-python3-runtime==4.9.* in /usr/local/lib/python3.10/dist-packages (from omegaconf>=2.0->effdet->layoutparser[layoutmodels,tesseract]->unstructured-inference==0.7.11->unstructured[all-docs]<0.11,>=0.10; extra == \"tools\" or extra == \"all\"->camel-ai[all]==0.1.6.8) (4.9.3)\n",
+            "Requirement already satisfied: matplotlib>=2.1.0 in /usr/local/lib/python3.10/dist-packages (from pycocotools>=2.0.2->effdet->layoutparser[layoutmodels,tesseract]->unstructured-inference==0.7.11->unstructured[all-docs]<0.11,>=0.10; extra == \"tools\" or extra == \"all\"->camel-ai[all]==0.1.6.8) (3.7.1)\n",
+            "Requirement already satisfied: contourpy>=1.0.1 in /usr/local/lib/python3.10/dist-packages (from matplotlib>=2.1.0->pycocotools>=2.0.2->effdet->layoutparser[layoutmodels,tesseract]->unstructured-inference==0.7.11->unstructured[all-docs]<0.11,>=0.10; extra == \"tools\" or extra == \"all\"->camel-ai[all]==0.1.6.8) (1.3.0)\n",
+            "Requirement already satisfied: cycler>=0.10 in /usr/local/lib/python3.10/dist-packages (from matplotlib>=2.1.0->pycocotools>=2.0.2->effdet->layoutparser[layoutmodels,tesseract]->unstructured-inference==0.7.11->unstructured[all-docs]<0.11,>=0.10; extra == \"tools\" or extra == \"all\"->camel-ai[all]==0.1.6.8) (0.12.1)\n",
+            "Requirement already satisfied: fonttools>=4.22.0 in /usr/local/lib/python3.10/dist-packages (from matplotlib>=2.1.0->pycocotools>=2.0.2->effdet->layoutparser[layoutmodels,tesseract]->unstructured-inference==0.7.11->unstructured[all-docs]<0.11,>=0.10; extra == \"tools\" or extra == \"all\"->camel-ai[all]==0.1.6.8) (4.53.1)\n",
+            "Requirement already satisfied: kiwisolver>=1.0.1 in /usr/local/lib/python3.10/dist-packages (from matplotlib>=2.1.0->pycocotools>=2.0.2->effdet->layoutparser[layoutmodels,tesseract]->unstructured-inference==0.7.11->unstructured[all-docs]<0.11,>=0.10; extra == \"tools\" or extra == \"all\"->camel-ai[all]==0.1.6.8) (1.4.5)\n"
+          ]
+        }
+      ],
+      "source": [
+        "pip install camel-ai[all]==0.1.6.8"
+      ]
```

Comment:
Do we need to install all dependencies here? It's a bit slow.

Reply:
Thanks @ZackYule ! Have updated with basic install